### ISSUE TITLE
ipfs files publish

### ISSUE THREAD

**Issue by lgierth** [ipfs/go-ipfs#3672]
The `ipfs files publish` command would take the current files root hash, and would publish it as an IPNS entry for the node. It'd effectively be a shortcut for `ipfs name publish $(ipfs files stat --hash /)`.

This seems trivial but I think it might remove UX friction.

**Comment by MikeFair** [comment]
Yes; that'd be aswesome; or written the other way (to take advantage of the different key):
ipfs name publish --files
ipfs name publish -k domain.key --files /domain.dom.directory

**Comment by MikeFair** [comment]
Then I just need ipfs add updated so I can do:
ipfs add --files /domain.dom.directory -r /path/to/root/of/domain/website
